packages/fuel-indexer/src/executor.rs

```diff
@@ -564,6 +602,7 @@ impl<F> Executor for NativeIndexExecutor<F>
 where
     F: Future<Output = IndexerResult<()>> + Send,
 {
+    /// Handle events for  native executor.
```

Comment:
Remove extra space here.

Reply:
I don't believe there's any whitespace here? This is a doc comment for line 606

Reply:
May be a bit of weirdness on my end then. 👍🏽